main.py

```diff
@@ -0,0 +1,39 @@
+name = input("What your name?")
+if name == "Lebron" or "Lebron James":
+    print("OH MY GOD MY GLORIOUS KING I LOVE YOU!")
+else:
+    print("Oh, ok loser.")
+
+age = int(input("How old are you?"))
+if age > 16:
+    print("Damn, old ahh jit. What was Rosa Parks really like?")
+if age < 16:
+    print("Damn, bro young ahh. You missed everything ngl. Fetus lookin ahh")
+if age == 16:
+    print("Real")
+
+food = input("What's your favorite food?")
+if food != "Chicken and Rice":
+    print("Sounds like someone needs to cut.")
+else:
+    print("Locked in.")
+
+color = input("What's your favorite color?")
+if color == "Blue" or "Red" or "Green":
+    print("Pretty basic ngl.")
+if color == "Black":
+    print("Okay edgelord.")
+if color == "Purple" or "Pink":
+    print("Bruh. Cornball lookin ahh.")
+if color == "White":
+    print("Based.")
+else:
+    print("Goofy art student ahh. Get a real job picasso.")
+
+sport = input("What's your favorite sport?")
+if sport == "Gooning":
+    print("Respectable")
+if sport == "Badminton":
+    print("Got it ON LOCK")
+else:
+    print("Goofy ahh playing that sploinky doinky")
```

Comment:
This is an interesting nuance. What you have here is two `if` checks, and if the user selects `Gooning`, then the output will be:

```
Respectable
Goofy ahh playing that spoinky doinky
```

The reason for this is `if sport == "Gooning"` is evaluating to true, and then when `if sport == "Badminton"` evaluates to false, it catches that `else` case. What it should look like, when checking multiple cases, is this:
```suggestion
if sport == "Gooning":
    print("Respectable")
elif sport == "Badminton":
    print("Got it ON LOCK")
else:
    print("Goofy ahh playing that sploinky doinky")
```

That `elif` basically means `else if`, which leaves this as one solid block (if one thing, else if another thing, otherwise our fallback).

Reply:
This one especially was helpful, I was wondering why "else if" wouldn't work, and I just assumed that python used "if" for all since I didn't get any syntax errors